docusaurus/docs/e2e-test-a-plugin/getting-started.md

```diff
@@ -0,0 +1,148 @@
+import ScaffoldPluginE2ERunTestsNPM from '@snippets/plugin-e2e-run-tests.npm.md';
+import ScaffoldPluginE2ERunTestsPNPM from '@snippets/plugin-e2e-run-tests.pnpm.md';
+import ScaffoldPluginE2ERunTestsYarn from '@snippets/plugin-e2e-run-tests.yarn.md';
+import ScaffoldPluginE2EDSWorkflowNPM from '@snippets/plugin-e2e-ds-workflow.npm.md';
+import ScaffoldPluginE2EDSWorkflowYarn from '@snippets/plugin-e2e-ds-workflow.yarn.md';
+import ScaffoldPluginE2EDSWorkflowPNPM from '@snippets/plugin-e2e-ds-workflow.pnpm.md';
+import ScaffoldPluginE2EProjectsConfigNPM from '@snippets/plugin-e2e-auth-project.npm.md';
+import ScaffoldPluginE2EProjectsConfigYarn from '@snippets/plugin-e2e-auth-project.yarn.md';
+import ScaffoldPluginE2EProjectsConfigPNPM from '@snippets/plugin-e2e-auth-project.pnpm.md';
+
+# Getting started
+
+This article will guide through how to install and configure `@grafana/plugin-e2e`, write tests and setup a basic Github workflow that can run your e2e tests targeting multiple versions of Grafana.
+
+## Prerequisites
+
+- You need to have a Grafana plugin [development environment](https://grafana.com/developers/plugin-tools/get-started/set-up-development-environment)
+- Node.js 18+
+- Docker
+- Basic Knowledge of Playwright. If you have not worked with Playwright before, we recommend following the [Getting started](https://playwright.dev/docs/intro) section in their documentation.
+
+### Installing Playwright
+
+Please refer to the [Playwright documentation](https://playwright.dev/docs/intro#installing-playwright) for instruction on how to install. `@grafana/plugin-e2e` extends Playwright APIs, so you need to have `Playwright/test` with a minimum version of 0.40.0 installed as a dev dependency in the package.json file of your plugin.
+
+## Set up `plugin-e2e`
+
+### Step 1: Installing @grafana/plugin-e2e
+
+Now open the terminal and run the following command in your plugin's project directory:
+
+<CodeSnippets
+snippets={[
+{ component: ScaffoldPluginE2InstallNPM, label: 'npm' },
+{ component: ScaffoldPluginE2EInstallYarn, label: 'yarn' },
+{ component: ScaffoldPluginE2EInstallPNPM, label: 'pnpm' }
+]}
+groupId="package-manager"
+queryString="current-package-manager"
+/>
+
+### Step 2: Configure Playwright
+
+Open the Playwright config file that was generated when Playwright was installed.
+
+1. Uncomment `baseUrl` and change it to `'http://localhost:3000'`.
+
+```ts title="playwright.config.ts"
+  baseURL: 'http://localhost:3000',
+```
+
+2. Playwright uses [projects](https://playwright.dev/docs/test-projects) to logically group tests that have the same configuration. We're going to add two projects. The first project called `auth` is a setup project will login to Grafana and store the authenticated state on disk. The second project `run-tests` runs all the tests in a browser of choice. By adding a dependency to the `auth` project we ensure login only happens once, and all tests in the `run-tests` project will start already authenticated.
```

Comment:
```suggestion
2. Playwright uses [projects](https://playwright.dev/docs/test-projects) to logically group tests that have the same configuration. We're going to add two projects:
   1. `auth` is a setup project will login to Grafana and store the authenticated state on disk.
   1. `run-tests` runs all the tests in a browser of choice. By adding a dependency to the `auth` project we ensure login only happens once, and all tests in the `run-tests` project will start already authenticated.
```

optional - think it improves readability

Reply:
Yep I like it!